pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -79,6 +79,13 @@ type PrometheusSpec struct {
 	Service  ServiceSpec `json:"service,omitempty"`
 	// +optional
 	ReserveDays int `json:"reserveDays,omitempty"`
```

Comment:
@Yisaer Not related to this pr, but I am curious about this field. So we only support retention with unit `d`? Not support other time like `h`?

Reply:
I think you are right. Maybe we would create a new proper property for this in the future.

Reply:
I have  support custom command ,so I think `reserveDays` or other time field is not necessary.